### ISSUE TITLE
[BUG] Release 1.3.9 integ-test fail 

### ISSUE THREAD

**Issue by jordarlu** [opensearch-project/opensearch-dashboards-functional-test#585]
### What is the bug?
**Release 1.3.9 integ-test fail** 

**OSD - tar fail** at functionalTestDashboards https://build.ci.opensearch.org/blue/organizations/jenkins/bundle-build-zhujiaxi/detail/bundle-build-zhujiaxi/123/pipeline
```
Couldn't determine Mocha version
(node:1228) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 preprocessor:close listeners added to [EventEmitter]. Use emitter.setMaxListeners() to increase limit
tput: No value for $TERM and no -T specified
error Command failed with exit code 8.
.....
.....
2023-03-14 04:29:10 ERROR    | functionalTestDashboards | with-security        | FAIL  |
2023-03-14 04:29:10 ERROR    | functionalTestDashboards | without-security     | FAIL  |
```
**OSD - rpm fail** at functionalTestDashboards https://build.ci.opensearch.org/blue/organizations/jenkins/bundle-build-zhujiaxi/detail/bundle-build-zhujiaxi/125/pipeline/65/
```
Couldn't determine Mocha version
(node:1436) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 preprocessor:close listeners added to [EventEmitter]. Use emitter.setMaxListeners() to increase limit
tput: No value for $TERM and no -T specified
error Command failed with exit code 7.
.....
.....
2023-03-14 06:19:54 ERROR    | functionalTestDashboards | with-security        | FAIL  |
2023-03-14 06:19:54 ERROR    | functionalTestDashboards | without-security     | FAIL  |
```
### How can one reproduce the bug?
_Steps to reproduce the behavior._

### What is the expected behavior?
_A clear and concise description of what you expected to happen._

### What is your host/environment?
_Operating system, version._

### Do you have any screenshots?
_If applicable, add screenshots to help explain your problem._

### Do you have any additional context?
_Add any other context about the problem._

**Comment by jordarlu** [comment]
https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/589

**Comment by joshuali925** [comment]
the PRs to address them is referenced above, thanks @jordarlu

**Comment by joshuali925** [comment]
i'll close this issue for now as PRs are merged and tests passed on my side for 1.3.9 using changes from those PRs. please reopen if they are still not working or there are other concerns
